pep-0646.rst

```diff
@@ -1045,14 +1081,14 @@ Implications
 As with the first grammar change, this change also has a number of side effects.
 In particular, the annotation of ``*args`` could be set to a starred object
 other than a ``TypeVarTuple`` - for example, the following nonsensical
-annotation is possible:
+annotation is possible, resulting in the following behaviour:
 
 ::
 
     >>> foo = [1, 2, 3]
-    >>> def bar(*args: *foo): pass  # Equivalent to `*args: tuple(foo)`
+    >>> def bar(*args: *foo): pass  # Roughly equivalent to `*args: next(iter(foo))`
```

Comment:
This would need to be updated as well (the example will fail if we use `UNPACK_SEQUENCE 1`).

Reply:
Fixed.